AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -100,6 +100,14 @@ extension ApplePayComponent {
 
             return paymentRequest
         }
+
+        @_spi(AdyenInternal)
+        public func updated(amount: Amount, localeIdentifier: String?) -> Self {
```

Comment:
Maybe `byUpdating(amount: .....)` more apple type of naming

Reply:
It returns new modified instance of struct, just like `.sorted()` or `.uppercased()` 🤔

Reply:
Agreed on `updating(with amount: localeIdentifier)`